src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -207,6 +209,16 @@ public static void SetDefaultGrabTouchAfterLeave(bool enable)
             defaultGrabTouchAfterLeave = enable;
         }
 
+        /// <summary>
+        /// If set to true, the <see cref="AllowOnlyOwnTouch"/> property value is set to true when all Views are created.
+        /// </summary>
+        /// <param name="enable">Sets value of AllowOnlyOwnTouch property</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static void SetDefaultAllowOnlyOwnTouch(bool enable)
```

Comment:
We might need some Config table for this kind of common default value set...
calling static method for changing default value is akward but this might be only choice for now....

Reply:
네 맞습니다.   이런류의 default 설정을 처음엔 theme같은 설정으로 포함시키려 했으나 theme으로는 만족시킬 수 가 없었으며
이런 default 설정을 할 수 있는 방안이 현재는 없어 부득이 static method로 추가하였습니다.